### PR TITLE
Fix instructions for installing nix

### PR DESCRIPTION
**Pull request by newhoggy**
As per https://github.com/input-output-hk/cardano-node/pull/1752

### CODE REVIEW

File: doc/getting-started/building-the-node-using-nix.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ than installing on Ubuntu/Debian.
 The [Nix Package Manager][nix] can be installed on most Linux distributions by downloading and
 running the installation script:
 ```
-curl https://nixos.org/nix/install > install-nix.sh
+curl -L https://nixos.org/nix/install > install-nix.sh
```
Comment (erikd) [inline doc/getting-started/building-the-node-using-nix.md:13]: We should uses the long version `--location` of the argument just for documentation sake.

**Review by erikd — CHANGES_REQUESTED** [review]

File: doc/getting-started/building-the-node-using-nix.md (hunk 1 above)
Comment (newhoggy) [reply doc/getting-started/building-the-node-using-nix.md:13]: This is how `nix` documents the install step.  It's probably best to follow their lead on this.

Comment (newhoggy) [reply doc/getting-started/building-the-node-using-nix.md:13]: image

**Comment by intricate** [comment]
Closing as duplicate of #1752

**Pull request closed without merging**